### PR TITLE
Add AIOZ Network Mainnet

### PR DESCRIPTION
**Pull request by trieuaioz**

### CODE REVIEW

File: osmosis-1/osmosis.zone_chains.json [hunk 1]
```diff
@@ -863,6 +863,18 @@
         "ibc-transfer",
         "ibc-go"
       ]
+    },
+    {
+      "chain_name": "aioz",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_chains.json:868]: Please define staking and fee rates (low, avg, high) in the [Chain Registry](https://github.com/cosmos/chain-registry/blob/master/aioz/chain.json)

**Comment by JeremyParish69** [comment]
Does a pool for AIOZ exist yet? Will need it for validation.

**Comment by JeremyParish69** [comment]
Any desire to proceed with this? Or should we close?

**Comment by trieuaioz** [comment]
Sorry for the late response, let's close it for now as I will get back later for the pool creation.

**Comment by JeremyParish69** [comment]
Closing for now as per submitter's request--expect to continue soon

**Pull request closed without merging**